assets/src/edit-story/components/library/panes/media/common/styles.js

```diff
@@ -36,16 +37,32 @@ export const PaneHeader = styled.div`
 `;
 
 export const MediaGalleryContainer = styled.div`
+  display: grid;
```

Comment:
Why do you need a grid here?

Reply:
Ah I guess we need it due to the current 2-column upload tab media gallery.

Reply:
It looks like this isn't needed. Removed.